composer.json

```diff
@@ -11,8 +11,9 @@
   },
   "config": {
     "sort-packages": true,
+    "process-timeout": 0,
     "platform": {
-      "php": "5.6.20"
```

Comment:
Maybe leave this to a separate pull request as we'll probably want to bump the versions of phpunit when doing this. The `phpunit-polyfills` are already present but we'll need to update the actual test files to match the new parent class signatures for the phpunit base classes. 

Overall, we could have this platform locked at the version we're using/expecting for all the dev tooling while the phpcompatability checker is used for ensuring that all code is compatible with the lowest version of PHP supported by the WP core, right?